docs/features/heat.rst

```diff
@@ -8,3 +8,4 @@ This section introduces the HEAT solver, a Tidy3D feature that solves the steady
     :maxdepth: 1
 
     ../../HeatSolver
+    ../../CPOHeat
```

Comment:
Isn't the usual policy for one notebook to go in one place only?

I'm not necessarily opposed to linking in multiple places just wondering.